tests/cases/seniors_app/case_seniors_app_schedule.yaml

```diff
@@ -386,4 +386,91 @@ TestFutureAppointmentCancelOrRescheduleButton:
   # Verify that the appointment appears in the schedule listing.
     - Type: wait_for
       Strategy: class_chain
-      Id: $PAGE.seniors_app_home.event_based_on_name_element$
\ No newline at end of file
+      Id: $PAGE.seniors_app_home.event_based_on_name_element$
+
+# CXTB-21: Verify Past appontment window (non-missed)
+TestNonMissedPastAppoinmentWindow:
```

Comment:
Also, in the name of the test, there's a typo, should be appointment

Reply:
fixed